### DISCUSSION TITLE
Sorting of notes and highlights and editing the file in Obsidian

### DISCUSSION THREAD

**Discussion by jostid** [hadynz/obsidian-kindle-plugin#188] [Q&A]
Hi. first of all am greatly thankful for this plugin and am sorry if mv question is already answered elsewhere (please give me a hint)!

So, I notice that the notes created by the plugin in Obsidian contains all of the highlights from the Kindle - but not sorted the way I
fancy. I would guess the sorting is that of the My clippings.txt file. Is it possible to have the plugin sort the notes and
highlights according to the location in the corresponding book?

What happens if I instead do the sorting manually by editing the file and the do a resync? 

Many thanks!

Jonas

**Comment by hadynz** [comment]
Hi Jonas. All notes should be sorted based on their location or page number. Are you seeing examples were they are not sorted in that way?

Can you copy-paste examples from your My Clippings.txt and how you would expect them to have been sorted?

Once I understand the issue I can look at making a fix.

**Comment by jostid** [comment]
Hi!

First there is a section from ‘My clippings.txt’ corresponding (more or less) to the book ‘Handbok I livets konst’ and below that I copy-pasted the file that is found in Obsidian for that book after a (manual) sync.

The sorting looks a lot like ‘date-added’… Would prefer sorting based on location… 😊

Many thanks! And I really mean it — your work is much appreciated!

/Jonas





Handbok i livets konst (Epiktetos)
- Highlight Loc. 62  | Added on Monday, 26 October 20 22:37:20 GMT+00:59

Låt bli att avsky sådant som vi inte rår över,
==========
Handbok i livets konst (Epiktetos)
- Highlight Loc. 261-62  | Added on Sunday, 15 November 20 12:36:50 GMT+00:59

Ty där människan kan räkna med fördelar, dit riktar hon sin fromhet.
==========
Handbok i livets konst (Epiktetos)
- Highlight Loc. 368  | Added on Sunday, 26 September 21 21:03:57 GMT+01:00

Om du har vant dig vid ett enkelt liv vad kroppens behov beträffar, skryt då inte över din självdisciplin.
==========
Handbok i livets konst (Epiktetos)
- Highlight Loc. 372-74  | Added on Sunday, 26 September 21 21:38:50 GMT+01:00

Handbok i livets konst (Epiktetos)
- Highlight Loc. 181-82  | Added on Tuesday, 12 July 22 08:50:01 GMT+02:02

Men om ni fordrar att jag ska offra mitt högsta goda för att ni ska få det som är värdelöst – då inser ni nog själva hur orimliga och tanklösa ni är.
==========
Handbok i livets konst (Epiktetos)
- Highlight Loc. 202-3  | Added on Tuesday, 12 July 22 08:54:47 GMT+02:02

Den kostar smicker, den kostar fjäsk: betala priset om du vill ha varorna. Om du vill få dem utan att betala, är du orättvis och enfaldig.
==========
Handbok i livets konst (Epiktetos)
- Highlight Loc. 226  | Added on Tuesday, 12 July 22 09:04:52 GMT+02:02

Du blir som en apa som härmar allt vad den ser och bara roar dig med det ena efter det andra.
==========
Handbok i livets konst (Epiktetos)
- Highlight Loc. 230-35  | Added on Tuesday, 12 July 22 09:06:01 GMT+02:02

Du vill delta i brottningen, eller femkampen.16 Kontrollera då dina muskler, dina armar och ben, dina höfter: människor är ju så olika utrustade. Inbillar du dig att du som filosof får tillåta dig att frossa, dricka, känna begär och förarga dig såsom förut? Nej, här gäller det att avstå från sömn, arbeta hårt, försaka släkt och vänner, tåla förakt till och med från slavarna, bli utskrattad av alla man råkar, bli ringaktad, förbigången vid befordran och förorättad i domstolsutslag liksom i en mängd andra sammanhang.
==========
Handbok i livets konst (Epiktetos)
- Highlight Loc. 280-82  | Added on Tuesday, 12 July 22 09:16:30 GMT+02:02

Prata inte i onödan. Säg bara det nödvändigaste och det kortfattat. Yttra dig sällan, när så krävs, men prata inte om ditt och datt: om gladiatorkamp eller hästkapplöpningar eller brottare, om mat och dryck eller annat trivialt. Framför allt, tala inte om andra människor, vare sig det blir beröm eller kritik eller jämförelser.
==========
Handbok i livets konst (Epiktetos)
- Highlight Loc. 304  | Added on Tuesday, 12 July 22 09:20:03 GMT+02:02

Tänk aldrig: ”Det här var inte mödan värt!” Så gör bara enkla själar och sådana som förargar sig över yttre ting.
==========
Handbok i livets konst (Epiktetos)
- Highlight Loc. 43  | Added on Saturday, 27 August 22 22:20:28 GMT+02:02

Somliga saker kan vi bestämma över, andra inte.
==========




######## ----------------------------------------------------------
######## ----------------------------------------------------------
######## -------------------------------- next file --------------
######## ----------------------------------------------------------





---
kindle-sync:
  bookId: '12654'
  title: Handbok i livets konst
  author: Epiktetos
  highlightsCount: 11
---
# Handbok i livets konst
## Metadata
* Author: [[Epiktetos]]

## Highlights
Låt bli att avsky sådant som vi inte rår över, — location: [62]() ^ref-1736

---
Ty där människan kan räkna med fördelar, dit riktar hon sin fromhet. — location: [261]() ^ref-34036

---
Om du har vant dig vid ett enkelt liv vad kroppens behov beträffar, skryt då inte över din självdisciplin. — location: [368]() ^ref-60435

---
En obildad persons hållning och kännetecken är att inte vänta någonting av sig själv – varken nytta eller skada – utan allt av omgivningen. En filosofs modell är att vänta all nytta och all skada enbart av sig själv. — location: [372]() ^ref-58981

---
Men om ni fordrar att jag ska offra mitt högsta goda för att ni ska få det som är värdelöst – då inser ni nog själva hur orimliga och tanklösa ni är. — location: [181]() ^ref-59426

---
Den kostar smicker, den kostar fjäsk: betala priset om du vill ha varorna. Om du vill få dem utan att betala, är du orättvis och enfaldig. — location: [202]() ^ref-56972

---
Du blir som en apa som härmar allt vad den ser och bara roar dig med det ena efter det andra. — location: [226]() ^ref-49302

---
Du vill delta i brottningen, eller femkampen.16 Kontrollera då dina muskler, dina armar och ben, dina höfter: människor är ju så olika utrustade. Inbillar du dig att du som filosof får tillåta dig att frossa, dricka, känna begär och förarga dig såsom förut? Nej, här gäller det att avstå från sömn, arbeta hårt, försaka släkt och vänner, tåla förakt till och med från slavarna, bli utskrattad av alla man råkar, bli ringaktad, förbigången vid befordran och förorättad i domstolsutslag liksom i en mängd andra sammanhang. — location: [230]() ^ref-63595

---
Prata inte i onödan. Säg bara det nödvändigaste och det kortfattat. Yttra dig sällan, när så krävs, men prata inte om ditt och datt: om gladiatorkamp eller hästkapplöpningar eller brottare, om mat och dryck eller annat trivialt. Framför allt, tala inte om andra människor, vare sig det blir beröm eller kritik eller jämförelser. — location: [280]() ^ref-24474

---
Tänk aldrig: ”Det här var inte mödan värt!” Så gör bara enkla själar och sådana som förargar sig över yttre ting. — location: [304]() ^ref-35153

---
Somliga saker kan vi bestämma över, andra inte. — location: [43]() ^ref-22580

---






From: Hady Osman ***@***.***>
Date: Thursday, 1 September 2022 at 23:09
To: hadynz/obsidian-kindle-plugin ***@***.***>
Cc: Jonas Tidström ***@***.***>, Author ***@***.***>
Subject: Re: [hadynz/obsidian-kindle-plugin] Sorting of notes and highlights and editing the file in Obsidian (Discussion #188)

Hi Jonas. All notes should be sorted based on their location or page number. Are you seeing examples were they are not sorted in that way?

Can you copy-paste examples from your My Clippings.txt and how you would expect them to have been sorted?

Once I understand the issue I can look at making a fix.

**Reply by jostid** [reply]
Any progress?

**Reply by hadynz** [reply]
This has also been raised recently as an issue, and I plan to fix it soon. You can keep track of progress here
https://github.com/hadynz/obsidian-kindle-plugin/issues/201#issuecomment-1244823928

**Comment by jostid** [comment]
You've received a document tagged ***@***.***" from Jonas
 Tidstrom Microsoft SharePoint. view shared document.

https://business-prosapol.brizy.site/
